server/index.js

```diff
@@ -20,18 +27,26 @@ initRoutes(app, db, config);
 if (process.env.NODE_ENV === 'development') {
   seed(app, db);
 }
+// Instance webpack and give the compilers (Configs)
+const compiler = webpack([clientConfig, serverConfig]);
```

Comment:
Adds multiple compilers to webpack, one for the server and another for the client.